server/src/userManagement.mjs

```diff
@@ -124,10 +128,19 @@ function associateUserWithWss(userId, wss) {
   user2wss.set(''+userId, wss);
 }
 
+//Associate each user with many ws connection,taking ws as an array of objects
 function associateUserWithWs(userId, ws) {
-  user2ws.set(''+userId, ws);
+  let wsArray = [];
```

Comment:
Same comment as above; Can either define this within the if block or just not assign a value here.

Reply:
wsArray is used in both if as well else block.Declared it as const

Reply:
done